### PR TITLE
Add the SecurityCoroutineContext class

### PR DESCRIPTION
**Pull request by mahkoCosmo**

### CODE REVIEW

**Review by vcarluer — APPROVED** [review]
LGTM

File: build.gradle.kts [hunk 1]
```diff
@@ -191,6 +191,7 @@ dependencies {
   implementation("org.springframework.boot:spring-boot-starter-aop")
 
   implementation("org.apache.tika:tika-core:${tikaVersion}")
+  implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.6.3-native-mt")
```
Comment (jreynard-code) [inline build.gradle.kts:194]: Could you please change for version 1.6.4 ? see [1.6.4](https://mvnrepository.com/artifact/org.jetbrains.kotlinx/kotlinx-coroutines-core/1.6.4)

**Review by jreynard-code — CHANGES_REQUESTED** [review]
LGTM!
Could you please upgrade the coroutines lib version ;)

**Review by jreynard-code — APPROVED** [review]
LGTM

**Pull request merged**